barretenberg/cpp/pil/vm2/precomputed.pil

```diff
@@ -14,4 +14,8 @@ pol constant sel_bitwise; // 1 in the first 3 * 256 rows.
 pol constant bitwise_op_id; // identifies if operation is AND/OR/XOR.
 pol constant bitwise_input_a; // column of all 8-bit numbers.
 pol constant bitwise_input_b; // column of all 8-bit numbers.
-pol constant bitwise_output; // output = a AND/OR/XOR b.
\ No newline at end of file
+pol constant bitwise_output; // output = a AND/OR/XOR b.
+
+// SHA256 Round Params Lookup
+pol constant sel_sha256_compression;
```

Comment:
I think you need to do tracegen for these
1. add to the precomputed trace
2. add it in the tracegen helper